### DISCUSSION TITLE
Does this skip the files you have already d/led on previous runs or does it d/l them again and overwrite?

### DISCUSSION THREAD

**Discussion by pauljb** [UltimaHoarder/UltimaScraper#907] [Q&A] [answered]
Don't want to draw any attention from OF, so does this skip previously d/led files?  Thanks
Sorry for dumb question.

**Comment by UltimaHoarder** [comment, marked as answer]
As long as you have the overwrite setting turned off in the config, it will skip files already marked as downloaded in the .db file.
